report_docx/models/ir_actions.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class IrActionsReportXml(models.Model):
+    _inherit = 'ir.actions.report.xml'
+
+    report_type = fields.Selection(
```

Comment:
Yes: don't redefine the selection list, using `selection_add =...` rather that `selection=...`